### PR TITLE
Add Anbox support

### PR DESCRIPTION
**Pull request by rubencarneiro**
anbox-tool report Kernel: OK

### CODE REVIEW

File: arch/arm64/configs/lineageos_bonito_defconfig [hunk 1]
```diff
@@ -1,3 +1,6 @@
+CONFIG_ANBOX=n
```
Comment (fredldotme) [inline arch/arm64/configs/lineageos_bonito_defconfig]: Remove this, it's not needed

File: arch/arm64/configs/lineageos_bonito_defconfig [hunk 2]
```diff
@@ -602,6 +605,8 @@ CONFIG_NVMEM=y
 CONFIG_QCOM_QFPROM=y
 CONFIG_SENSORS_SSC=y
 CONFIG_MSM_TZ_LOG=y
+# CONFIG_ANBOX_BINDER_IPC=y
+# CONFIG_ANBOX_BINDER_IPC_32BIT is not set
```
Comment (fredldotme) [inline arch/arm64/configs/lineageos_bonito_defconfig]: Also not used.

File: arch/arm64/configs/lineageos_bonito_defconfig [hunk 3]
```diff
@@ -1,3 +1,6 @@
+CONFIG_ANBOX=n
+CONFIG_ANDROID_BINDER_DEVICES="binder,hwbinder,vndbinder,anbox-binder,binder1"
```
Comment (fredldotme) [inline arch/arm64/configs/lineageos_bonito_defconfig]: Move that below CONFIG_ANDROID_BINDER_IPC=y

File: arch/arm64/configs/lineageos_bonito_defconfig [hunk 4]
```diff
@@ -1,3 +1,6 @@
...
+CONFIG_MEMCG_KMEM=y
```
Comment (fredldotme) [inline arch/arm64/configs/lineageos_bonito_defconfig]: Move that below CONFIG_MEMCG=y

**Review by fredldotme — COMMENTED** [review]
Added some comments, please look into them.

**Comment by rubencarneiro** [comment]
Type error Moved GROUPED DEFINITIONS for ANBOX and KMEM

**Review by fredldotme — APPROVED** [review]

**Pull request merged**